s3_upload/utils/log.py

```
@@ -35,31 +39,82 @@ def set_file_handler(logger, log_dir) -> logging.Logger:
-        [isinstance(x, TimedRotatingFileHandler) for x in logger.handlers]
-    ) and os.path.exists(log_file):
-        # log file handler already set and log file exists => use it
-        logger.info("Log file handler already set to %s", log_file)
+    if existing_file_handler:
+        logger.debug("FileHandler already set")
         return logger
 
-    logger.info(
-        "Initialised log fileHandler, setting log output to %s", log_file
+    log_file = os.path.join(
+        log_dir, f"s3_upload.log.{datetime.now().strftime('%Y-%m-%d')}"
```

Ruff (0.8.0)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)